### PR TITLE
OPENE-533 files list made common and added to organization edit

### PR DESCRIPTION
**Pull request by ArnasPetraitis**

### CODE REVIEW

File: app/src/contacts/OrganizationController.js [hunk 1]
```diff
@@ -2,36 +2,65 @@
...
+        contactsService.getOrganization($stateParams.storeProtocol, $stateParams.storeIdentifier, $stateParams.uuid).then(function(organization) {
+            vm.organization = organization;
+        });
     }
 
-    function loadOrganizations(){
-        contactsService.getOrganizations(vm.searchQuery).then(function(result){
-            vm.organizations = result.items;
+    function initFiles() {
+        $scope.filesVm = {};
```
Comment (jrudinskij) [inline app/src/contacts/OrganizationController.js]: Seems like would be better to extract this controller into separate OrganizationFilesController.

File: app/src/files/FilesController.js [hunk 2]
```diff
@@ -66,9 +68,18 @@ function FilesController($scope, $injector, filesService, $translate, $mdDialog,
             templateUrl: 'app/src/files/view/addFiles.html',
             parent: angular.element(document.body),
             targetEvent: ev,
-            clickOutsideToClose: true
+            clickOutsideToClose: true,
+            locals: {
+                params: {
```
Comment (jrudinskij) [inline app/src/files/FilesController.js]: I guess would be better call it filesUploadService or similar instead of params.

**Comment by ArnasPetraitis** [comment]
retest this please

**Comment by ArnasPetraitis** [comment]
retest this please

**Pull request merged**